api/src/routes/games/translations/Translations.ts

```diff
@@ -0,0 +1,184 @@
+import { Router } from 'express';
+import { prisma } from '../../../database/Database';
+import { Prisma } from '@prisma/client';
+
+const translationsRouter = Router();
+
+translationsRouter.get('/:slug/translations', async (req, res) => {
+    const { slug } = req.params;
+
+    const result = (await prisma.game.findUnique({ where: { slug } }))
+        ?.translations;
+    res.json(result);
+    return;
+});
+
+translationsRouter.post('/:slug/translations', async (req, res) => {
```

Comment:
Should this post and the delete actually be a single PUT? I know they don't have precisely the same logic, but given that we're not indexing into the array for deletes or updates, I'm inclined to think that requiring the entire array to be resented when making changes may actually be reasonable here to help reduce the overhead of these functions, which are getting to be a little awful